go/cmd/dolt/commands/sql.go

```diff
@@ -224,18 +224,22 @@ func (cmd SqlCmd) Exec(ctx context.Context, commandStr string, args []string, dE
 
 		// Look in data directory (which is necessarily current directory) for doltcfg
 		path = filepath.Join(dataDir, DefaultCfgDirName)
```

Comment:
The way this is written is confusing. Use variable names to refer to actual directories, e.g. `currDirCfg`, `parentDirCfg`

Don't reuse path for these two different values, and don't assign anything to cfgDirPath (which exists outside this condition) until you decide what value it should have